network/index.js

```diff
@@ -18,18 +18,31 @@ import { useState, useEffect } from 'react';
 
 let unsupported;
 
+
 const useNetworkStatus = initialEffectiveConnectionType => {
-  if (typeof navigator !== 'undefined' && 'connection' in navigator && 'effectiveType' in navigator.connection) {
+  if (
+    typeof navigator !== 'undefined' &&
+    'connection' in navigator &&
+    'effectiveType' in navigator.connection
+  ) {
     unsupported = false;
   } else {
     unsupported = true;
   }
 
+  const getEffectiveConnectionType = () => {
+    const isOnline = 'onLine' in navigator ? navigator.onLine : true;
```

Comment:
@JCofman 

What if `navigator` was not supported?
According to https://caniuse.com/#search=navigator.onLine, it's widely supported. But as we have `unsupport` check like above, it could make more sense if we added that check.

https://github.com/rehooks/online-status/blob/master/src/index.js
This could be helpful.